docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+# Release 352 (XX YYY 2021)
+
+## General
+
+* Add support for [`WINDOW` clause](window_clause). ({issue}`651`)
+* Add {func}`version` function. ({issue}`4627`)
+* Allow prepared statement parameters for `SHOW STATS`. ({issue}`6582`)
+* Update tzdata version to 2020d. As a result, queries can no longer reference the 
+  `US/Pacific-New` zone, as it has been removed. ({issue}`6660`)
+* Improve performance of queries using {func}`rank()` window function. ({issue}`6333`)
+* Improve performance of {func}`sum` and {func}`avg` for `decimal` types. ({issue}`6951`)
+* Improve join performance. ({issue}`5981`)
+* Improve query planning time. ({issue}`6544`)
```

Comment:
Some more specifics when this one applies? Would adding sth like "for queries using range predicates" make sense? @findepi

Reply:
Good catch. It's actually applicable more when using large IN lists, or when joining on hive partitioning column, which implies large IN list.
Also may improve DF handling, but i didn't measure that.